### DISCUSSION TITLE
Are you guys open to adding more backend?

### DISCUSSION THREAD

**Discussion by baseplate-admin** [eylles/pywal16#43] [Ideas]
Hi i have a question. Are you folks willing to add more backends to `pywal`?

**Comment by eylles** [comment]
yeh, in principle it should be easy.

**Reply by baseplate-admin** [reply]
If i add support for `modern_colorthief`, are you guys willing to accept that?

`modern_colorthief` is `colorthief` written in rust and it provides a small binary compared to `fast-colorthief`, while being faster than `fast-colorthief` in some scenarios

**Reply by eylles** [reply]
yeh, if the binary is compatible with either colorthief or fast-colorthief it would be just a matter of copying the backend file of the closest one and modifiying it to use the modern_colorthief binary.
